.version

```diff
@@ -1 +1 @@
-0.2.0
\ No newline at end of file
+0.3.0
```

Comment:
can we call this `0.2.1`? I don't think we need a major version bump for this.

Reply:
It's a minor bump. I think we should because this is actually a breaking change.

Reply:
The `this` referencing the previous two commits where the change was actually made.

Reply:
Please update the README, so the example still works.

Reply:
Now that you mention that, "0.3.0" is not a major change, it is a minor according to semver of course (MAJOR.MINOR.PATCH) https://semver.org/

There is a lot of discussion around allowing or not breaking changes in minors when major is 0 but given we are doing it, I would stick to the pattern of: minor can have breaking changes, patch can't.

Reply:
That's what I said - "It's a minor bump" :) Yes, given that this is a pre-release gem I think we are within limits to make a breaking change which only updates the minor, as you can't change the major without making it production-ready.